src/test/RecordingMetadata.tsx/LabelCell.test.tsx

```diff
@@ -115,25 +116,27 @@ describe('<LabelCell />', () => {
     );
 
     let count = 0;
-    mockLabelStringDisplayList.forEach((labelAsString, index) => {
+    let index = 0;
+    for (const labelAsString of mockLabelStringDisplayList) {
```

Comment:
Just for reference, why does this need to be changed?

Reply:
Ah I was not sure if forEach support async in callback. So, I just used loop just in case.